src/term.js

```diff
@@ -1225,13 +1225,13 @@ Terminal.prototype.refresh = function(start, end) {
 
     attr = this.defAttr;
     i = 0;
-
-    for (; i < width; i++) {
+    var widthd = width;
+    for (; i < widthd; i++) {
       data = line[i][0];
       ch = line[i][1];
 
       if (i === x) data = -1;
-
+      if (ch.charCodeAt()>=127){widthd=widthd-0.7}
```

Comment:
127 and 0.7 also my empirical data, so maybe just fit for me

thanks for your work!